templates/setting_list.mustache

```diff
@@ -41,7 +41,7 @@
     Setting configtext.
 }}
 <div class="form-text defaultsnext">
-    <input list="{{name}}" name="{{name}}" value="{{value}}" size="{{size}}" id="{{id}}" class="form-control {{#forceltr}}text-ltr{{/forceltr}}">
+    <input type="text" list="{{name}}" name="{{name}}" value="{{value}}" size="{{size}}" id="{{id}}" class="form-control {{#forceltr}}text-ltr{{/forceltr}}"/>
```

Comment:
This change is needed to fix input appearance:

![image](https://user-images.githubusercontent.com/22066290/84107616-7a61c980-aa61-11ea-92cd-b92fd78518af.png)